mixer/pkg/protobuf/yaml/dynamic/handler.go

```diff
@@ -92,11 +97,19 @@ func BuildHandler(name string, connConfig *policypb.Connection, sessionBased boo
 		return nil, errors.Errorf("empty connection address")
 	}
 
+	timeout := time.Duration(0)
+	if connConfig.GetTimeout() != nil {
+		timeout, err = ptypes.DurationFromProto(connConfig.GetTimeout())
```

Comment:
non-blocking: should we have used a gogo annotation on this proto to automatically handle this conversion?

see: https://github.com/gogo/protobuf/blob/master/test/types/types.proto#L82

Reply:
Thanks for the linked! Created istio/api#673 and we can wait for that to be checked in then update it here. master branch is locked anyway.